src/app/cube/details/components/action-panel/editorial-action-pad/editorial-action-pad.component.html

```diff
@@ -23,11 +23,11 @@
 </button>
 
 <button
-  *ngIf="assignEvaluators"
+  *ngIf="makeRevision"
```

Comment:
Does this appear for a mapper as well?

Reply:
It wasn't but I moved it out of the editorial component.  Now anyone with an access group can access it.

Reply:
Thank you!